source/includes/administration/_quotas.md

```diff
@@ -0,0 +1,76 @@
+## Quotas
+
+Quotas are used to limit the resource usage on a service connection consumed by an organization. The primary use case for CloudMC is to enforce limited quotas on trial accounts. Quotas can target different resources. For example, number of vCPUs, allocated memory, disk size, network bandwidth, number of public IPs and more.
+
+<!------------------- LIST QUOTAS --------------------->
+
+### List quotas
```

Comment:
There should also be a section for `get /quotas/:id`

Reply:
- [ ] Add a section for retrieving a single quota.